spec/interfaceToC.dd

```diff
@@ -152,6 +152,7 @@ $(H2 $(LNAME2 data_type_compat, Data Type Compatibility))
 
         $(TROW3 core.stdc.config.c_long, long, long)
         $(TROW3 core.stdc.config.c_ulong, unsigned long, unsigned long)
+        $(TROW2 core.stdc.stdint.intptr_t, intptr_t, $(D uintptr_t))
```

Comment:
You are putting both on the same row, but what is actually needed is:

```
 $(TROW2 core.stdc.stdint.intptr_t, intptr_t, intptr_t)
 $(TROW2 core.stdc.stdint.uintptr_t, uintptr_t, uintptr_t)
```

Reply:
you still need to delete this row.

Reply:
I did it, thank you. I hope it is ok now.